modules/packages/LinearAlgebra.chpl

```diff
@@ -312,17 +312,16 @@ proc _array.T where this.domain.rank == 1 { return transpose(this); }
 
 */
 proc transpose(A: [?Dom] ?eltType) where Dom.rank == 2 {
```

Comment:
If we never intend for users to call this with a distributed array, it would be good to add a compilerError if ``Dom`` is distributed. This could just be a TODO for now.